website/content/en/preview/troubleshooting.md

```diff
@@ -179,6 +179,17 @@ approach, and now it's much more restrictive.
 
 ## Provisioning
 
+### Instances with swap volumes fail to register with control plane
```

Comment:
Do we want to back-port this change across all versions if this can occur everywhere?

Reply:
I agree, although I have only seen this behaviour in limited testing.